script/Character.py

```diff
@@ -1,36 +1,45 @@
 # Program imports
 import random
 
-from script import Items
+import Items
 from Mechanics.ui_mechanics import *
 from Map_Gen.ParseMap import WorldMap
 
 
 class Character:
-    def __init__(self, name, max_health, melee_attack, magic_attack,
-                 max_mana, max_stamina, defense, luck, level=1):
+    def __init__(self, name=None, max_health=1, health=max_health, melee_attack=None, magic_attack=None,
```

Comment:
Mentioned on discord, but I'm not sure whether this 'health=max_health' default behaves exactly as we expect it to.

Reply:
You are right to question this, This does not work. You cannot initiate a value from another value that hasn't already been initiated before this call. In the testCharacter.py, this has been corrected to have health=1 for the time being. Again, we do not have functional code in Character.py. right now we need to get it functional. We can change default values later.